ui-participant/src/hub/HubPage.tsx

```diff
@@ -93,6 +94,7 @@ const StudySection = (props: StudySectionProps) => {
   return (
     <>
       <h1 className="mb-4">{matchedStudy.name}</h1>
+      {enrollee.kitRequests.length > 0 && <KitTask kitRequests={enrollee.kitRequests} />}
```

Comment:
it seems odd just from a naming perspective to pass a list of requests to a component called KitTask.  (I was expecting it to take a ParticipantTask as an arugment).  Maybe rename 'KitTask' to 'KitActions' or 'KitBanner' or something like that?